Devscord.DiscordFramework/Commons/Exceptions/InvalidArgumentsException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Devscord.DiscordFramework.Commons.Exceptions
+{
+    public class InvalidArgumentsException : Exception
```

Comment:
Dodaj do Resources jeszcze odpowiednią wiadomość dla tego wyjątku

Reply:
jak to zrobić?

Reply:
![image](https://user-images.githubusercontent.com/35143425/83351681-7db0d300-a346-11ea-85a2-68d4d7759c71.png)
W tym pliku resources dodaj nazwę wyjątku i odpowiedni komunikat dla niego (tak jak mają inne wyjątki)

Reply:
nie trzeba go czasem jeszcze dodać do LogException w ExceptionHandlerService?

Reply:
Tak, musisz dodać tam wyświetlanie odpowiedniej wiadomości dla tego wyjątku, tak jak w przypadku pozostałych